src/d/actor/d_a_tag_kk1.cpp

```diff
@@ -3,64 +3,120 @@
 // Translation Unit: d_a_tag_kk1.cpp
 //
 
+
 #include "d/actor/d_a_tag_kk1.h"
+#include "d/d_com_inf_game.h"
 #include "d/d_procname.h"
 
-/* 000000EC-00000120       .text __ct__15daTag_Kk1_HIO_cFv */
+
+static daTag_Kk1_HIO_c l_HIO;
+
+static const u8 dummy5[] = { 0x00, 0xFF, 0x00, 0x80};
+
 daTag_Kk1_HIO_c::daTag_Kk1_HIO_c() {
-    /* Nonmatching */
+    static f32 a_prm_tbl[] = {350.0f,30.0f,0.0f};
+    mHorizontalDistance = a_prm_tbl[0];
+    mVerticalDistance = a_prm_tbl[1];
+    mUnusedU8 = *reinterpret_cast<u8*>(&a_prm_tbl[2]); //TODO: This u8 appears to be inside the float table. May be a better way to refactor.
+    mNo = -1;
 }
 
 /* 00000120-00000128       .text createInit__11daTag_Kk1_cFv */
-void daTag_Kk1_c::createInit() {
-    /* Nonmatching */
+BOOL daTag_Kk1_c::createInit() {
+    return TRUE;
 }
 
 /* 00000128-00000130       .text _draw__11daTag_Kk1_cFv */
 bool daTag_Kk1_c::_draw() {
-    /* Nonmatching */
+    return true;
 }
 
 /* 00000130-0000024C       .text _execute__11daTag_Kk1_cFv */
 bool daTag_Kk1_c::_execute() {
-    /* Nonmatching */
+
+    cXyz* player_pos = &dComIfGp_getPlayer(0)->current.pos; 
+    f32 distance = std::sqrtf(current.pos.abs2(*player_pos));
+    f32 vert_distance = dComIfGp_getPlayer(0)->current.pos.y - this->current.pos.y;
```

Comment:
Explicit `this->` not required
```suggestion
    f32 vert_distance = dComIfGp_getPlayer(0)->current.pos.y - current.pos.y;
```